spec/requests/item_presenter_spec.rb

```diff
@@ -2,22 +2,22 @@
 
 describe Api::V1::ItemPresenter do
   describe ".to_json" do
-    it "returns an item object with only the id, name, sellIn, and quality attributes" do
-      item = FactoryBot.create(:item)
-
+    it "returns an item object with the id, name, sellIn, description, and quality attributes" do
+      item = FactoryBot.create(:item, name: "Foo", sell_in: 5, quality: 11, description: "Bar")
       presented_item = Api::V1::ItemPresenter.to_json(item)
 
+      expect(presented_item).to include(:id)
       expect(presented_item).to include(:name)
       expect(presented_item).to include(:sellIn)
       expect(presented_item).to include(:quality)
+      expect(presented_item).to include(:description)
       expect(presented_item).not_to include(:created_at)
       expect(presented_item).not_to include(:updated_at)
     end
   end
 
   it "returns item with camelCase attributes" do
-    item = FactoryBot.create(:item, name: "FooBar", sell_in: 5, quality: 11)
-
+    item = FactoryBot.create(:item, name: "Foo", sell_in: 5, quality: 11, description: "Bar")
```

Comment:
See previous comments regarding style. One thing I've learned from others is that it's often easier to read code that is more vertical than horizontal.